fast/stages/2-networking-b-nva/net-prod.tf

```diff
@@ -15,11 +15,21 @@
  */
 
 # tfdoc:file:description Production spoke VPC and related resources.
+locals {
+  nva_load_balancers = (var.network_mode == "ncc_ra") ? null : {
```

Comment:
This is really opaque. 
I would instead define two locals (e.g. `_simple_nva_lb` and `_regional_nva_lb`, and conditionally assign their value to `nva_load_balancers`.
As an alternative write a verbose comment to what is happening there 😆

Reply:
fixed